Source/JavaScriptCore/runtime/IntlDurationFormat.cpp

```diff
@@ -304,7 +304,7 @@ struct Element {
     ElementType m_type;
     TemporalUnit m_unit;
     String m_string;
-    double m_value;
+    bool m_valueSign;
```

Comment:
Can we name this `m_isValueNegative` or `m_valueSignBit`? `m_valueSign` doesn't quite indicate what true means. Also, we should probably put this after `m_type` since that packs the struct a bit smaller.

Reply:
Thank you, I've addressed your comment.